dev/VSIX/ProjectTemplates/Neutral/CppWinRT/RuntimeComponent/Class.cpp

```diff
@@ -1,24 +1,10 @@
-// Copyright (c) Microsoft Corporation and Contributors.
-// Licensed under the MIT License.
-
 #include "pch.h"
 #include "Class.h"
-#if __has_include("Class.g.cpp")
```

Comment:
Keep this, as simple classes might not have a public constructor

Reply:
I also realise that users can opt out of component optimisations, which gets rid of the g.cpp file.